r/tools/nixlibs.R

```diff
@@ -79,6 +79,10 @@ find_latest_nightly <- function(description_version,
 }
 
 try_download <- function(from_url, to_file, hush = quietly) {
+  # We download some fairly large files, so ensure the timeout is set appropriately
+  opts <- options(timeout = max(300, getOption("timeout")))
```

Comment:
How big are our binaries? We could back of the envelope math for downloading that in worst-case speed. You might have already done this calculation, if so, would be nice to add that to the comment so we know

Reply:
That's wonderful. Thank you!